README.de.md

```diff
@@ -70,7 +70,11 @@ MemFree ist mit leistungsstarken Funktionen ausgestattet, die verschiedene Such-
 
 -   🌍**Mehrsprachige Unterstützung**: Verfügbar in Englisch, Chinesisch, Deutsch, Französisch, Spanisch, Japanisch und Arabisch.
 
+<<<<<<< HEAD
+-   🔗**Chrome-Lesezeichen-Synchronisierung**: One-click synchronization and indexing.
+=======
 -   🔗**Chrome-Lesezeichen-Synchronisierung**: Synchronisierung und Indizierung mit einem Klick.
+>>>>>>> cc296ef9fbf7d399a15edfe9b9fda3f7dc086433
 
```

Comment:
you could run `pull main --rebase`, then reslove the conflict，then push a commit to this branch